ui/src/api/config.js

```diff
@@ -2,5 +2,9 @@
 
 export default {
   baseURL: VUE_APP_API_BASE_URL,
-  timeout: 30000
+  timeout: 30000,
+  common: {
+    Accept: "application/json",
+    "Content-Type": "application/json"
+  }
```

Comment:
Is it possible to make the key format consistent, i.e. Accept and Content-Type , or "Accept" and "Content-Type"?

Reply:
Nope XD its linted by eslint. " is added because the key contains "-".